base/docs/basedocs.jl

```diff
@@ -789,6 +789,42 @@ run(pipeline(`cat test.txt`, devnull))
 """
 devnull
 
+""" 
+    `\$`
+The `\$` before a variable will interpolate values into string literals.  Also constructs `Expr`
```

Comment:
"A" instead of "The"? "interpolates" instead of "will interpolate"? Add "quoted expressions, and commands (``[`Cmd`](@ref)``)" after "string literals" perhaps (and by doing so perhaps the second sentence can be dropped)?